### PR TITLE
Support rgw

### PR DESCRIPTION
**Pull request by mnasiadka**
based on #17

### CODE REVIEW

**Comment by cityofships** [comment]
New inventory group should be mentioned in the README

**Review by cityofships — APPROVED** [review]

File: roles/cephadm/templates/cluster.yml.j2 [hunk 1]
```diff
@@ -12,6 +12,9 @@ labels:
 {% if host in groups['osds'] %}
 - osd
 {% endif %}
+{% if host in groups['rgws'] %}
```
Comment (markgoddard) [inline roles/cephadm/templates/cluster.yml.j2:15]: Unconditional reference - group is not optional.

File: roles/cephadm/templates/cluster.yml.j2 [hunk 2]
```diff
@@ -25,3 +28,15 @@ placement:
 service_type: crash
 placement:
   host_pattern: "*"
+---
+{% if groups['rgws'] | length > 0 %}
```
Comment (markgoddard) [inline roles/cephadm/templates/cluster.yml.j2:32]: ditto

File: roles/cephadm/defaults/main.yml [hunk 3]
```diff
@@ -27,3 +27,5 @@ cephadm_cluster_network: ""
 # OSDs
 cephadm_osd_devices: []
 cephadm_osd_spec: []
+# RADOSGW
+cephadm_radosgw_services: []
```
Comment (markgoddard) [inline roles/cephadm/defaults/main.yml:31]: Format should be documented. It also seems to be inconsistent with how OSDs are configured.

**Pull request merged**